lib/capybara-screenshot/saver.rb

```diff
@@ -26,11 +26,15 @@ def initialize(capybara, page, html_save=true, filename_prefix='screenshot')
       end
 
       def save
-        # if current_path empty then nothing to screen shot as browser has not loaded any URL
-        return if page.current_path.to_s.empty?
-
-        save_html if @html_save
-        save_screenshot
+        begin
+          # the current_path may raise error with selenium
+          if !page.current_path.to_s.empty?
+            save_html if @html_save
+            save_screenshot
+          end
+        rescue StandardError => e
```

Comment:
I dislike the block here that catches any save_html or save_screenshot exception now as a blanket catch, which is a nasty side effect of what we wanted.

Could you instead do something more like this please:

```
dev save
  # if current_path empty then nothing to screen shot as browser has not loaded any URL
  return if current_path.empty?

  # ... leave rest as it was
end

private
dev current_path
  page.current_path.to_s
rescue StandardError => e
  warn "WARN: Screenshot could not be saved. `page.current_path` raised exception: #{e}."
end
```